proto/api/v1/process_baseline_service.proto

```diff
@@ -60,6 +60,14 @@ service ProcessBaselineService {
             };
     }
 
+    // `GetProcessBaselineById` returns the single
+    // process baseline referenced by the given ID.
+    rpc GetLoadProcessBaseline (GetProcessBaselineRequest) returns (storage.ProcessBaseline) {
+        option (google.api.http) = {
+            get: "/v1/processbaselines/lazy/key"
```

Comment:
we need an endpoint that will create a baseline if one doesn't exist from the UI without changing how things work for the user.  I also feel that the endpoint that simply does a get is of value as well even if it just helps us in testing.

Reply:
We do not plan to change existing interface.
If we add a endpoint here, we are changing two things in API interface:
1. Added GetLoadProcessBaseline
2. Changed GetProcessBaseline cannot get process baseline in observation period.
I would prefer to keep existing API until we have enough reason to change it.

Reply:
Probably need @connorgorman to weigh in when he gets back to reviewing this.  One of the main tenets of this idea was that the user could view a baseline if they chose which would create one at that point in time.  

Doing it the way I did hides the details from the user.  They ask for a baseline and can see one.  It doesn't matter if we have been building one as we go or we just built it on their request.  Functionality doesn't change from their perspective.  I would imagine changing it so they couldn't view a baseline until observation ended would potentially open a new can of worms with documentation and everything else.

Reply:
I'm not sure we need a new endpoint. I think we can do the following:

If we call get baseline, but the deployment doesn't exist and baseline doesn't exist then we return not found
if we call get baseline and the deployment does exist, but the baseline does not because we are in observation then we can build it from the existing process
if we call get baseline and the baseline exists, then we just return it regardless of deployment existing